NodeEdge/wrap_edge_binary

```diff
@@ -6,7 +6,7 @@ mv "$WRAPPER_PATH" "$BASE_PATH"
 
 cat > "$WRAPPER_PATH" <<_EOF
 #!/bin/bash
-
+umask 002
```

Comment:
Can we leave a comment explaining the reasoning of the command placed here?

Reply:
@diemol, I have to confess that I have not tested this change with Edge as it is not used in our environment.